### PR TITLE
Image support

### PR DESCRIPTION
**Pull request by karlding**
Add image support! I'm disabling the unit tests for now, since we'll probably have to rethink how that's going to work. Also, switched from using `mutt` to `sendmail`.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -10,8 +10,8 @@ addons:
       - shellcheck
 
 script:
-  - shellcheck *.sh src/*.sh
-  - tests/run_all.sh
+  - shellcheck **/*.sh
+  #- tests/run_all.sh
```
Comment (Caelink) [inline .travis.yml]: Gonna branch off your branch and see if I can't get unit tests working before merge if possible

File: src/config.sh [hunk 2]
```diff
@@ -3,3 +3,7 @@
 export EMAILS="${HOME}/.local/share/lucashi"
 export MAIL_SCRIPT="$RUN_DIR/src/disrupt.sh"
 export FACTS_SOURCE="$RUN_DIR/assets/lucashifacts"
+
+# controls how likely a lucashi image will be shown
+# p = 1 / IMAGE_RARITY
```
Comment (Caelink) [inline src/config.sh:8]: I guess the config file should be v well documented

File: src/disrupt.sh [hunk 3]
```diff
@@ -1,15 +1,35 @@
...
+    else
+        echo '--body--' >> "${TEMP_FILE}"
+    fi
+    echo "--message--" >> "${TEMP_FILE}"
+
+    /usr/sbin/sendmail "${EMAIL_ADDRESS}" < "${TEMP_FILE}"
+    rm "${TEMP_FILE}"
 else
-    echo "$tmpstring"
+    echo "TODO"
```
Comment (Caelink) [inline src/disrupt.sh]: Can probably mimic the behaviour by building the tmp file no matter what, and simply echoing if not given an email

**Review by Caelink — COMMENTED** [review]
Seems fine, lemme see if I can't make the unit test work

File: src/disrupt.sh (hunk 3 above)
Comment (karlding) [reply src/disrupt.sh]: o trü.. we can probably override the dice rolls and have it use a specific image as well in the tests

Comment (Caelink) [reply src/disrupt.sh]: yeah just in your test config give it a chance of 1/1

File: src/disrupt.sh [hunk 4]
```diff
@@ -1,15 +1,38 @@
...
+
+
+# add lucashifact
+cp "${MSG_TEMPLATE_BASE}" "${TEMP_FILE}"
+echo "${RANDOM_FACT_HTML}" >> "${TEMP_FILE}"
+echo '</p>' >> "${TEMP_FILE}"
+
+# add the image if necessary
+if [ "$DICE_ROLL" -eq 1 ]; then
+    RANDOM_IMAGE=$(find "${IMG_ASSET_DIR}" -type f | shuf -n 1)
```
Comment (karlding) [inline src/disrupt.sh]: Should I add ``lucas.zip`` as part of this PR?

Regardless, we probably will need some sort of error-checking to handle the case when ``${IMG_ASSET_DIR}`` doesn't exist, or doesn't contain any images.

Comment (karlding) [inline src/disrupt.sh]: Should I add ``lucas.zip`` as part of this PR?

Regardless, I'll add some error-checking to handle the case when ``${IMG_ASSET_DIR}`` doesn't exist, or doesn't contain any images.

File: .travis.yml [hunk 5]
```diff
@@ -10,7 +10,7 @@ addons:
       - shellcheck
 
 script:
-  - shellcheck *.sh src/*.sh
+  - shellcheck **/*.sh
```
Comment (Caelink) [inline .travis.yml:13]: 🎉

**Review by Caelink — APPROVED** [review]
Seems fine to me yo 💯 Gonna branch off this and try to associate images with facts if I feel like procrastinating

**Comment by karlding** [comment]
Yeah. It should be pretty easy to support different image types, it looks like 

```bash
file -b -i filename
```

should do the trick

**Pull request merged**